dags/analyzer_helper/tests/integration/test_discord_extract_raw_info.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+from datetime import datetime
+
+from dags.analyzer_helper.discord.discord_extract_raw_infos import DiscordExtractRawInfos
+from dags.hivemind_etl_helpers.src.utils.mongo import MongoSingleton
+
+
+class TestDiscordExtractRawInfos(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
There's a class for setUp and tearDown of `unittest.TestCase`

Writing the classes below will do the trick without needing to use `classmethod` decorator

```
# the class that always would run before each test function
def setUp(self):
    pass

# the class that always would run after each test function
def tearDown(self):
    pass
```

What's your opinion on using these instead of defining classmethod for tearDown and setUp?

Reply:
Given that the operations we are performing here relatively lightweight, I think you are right. In case that tests become time consuming we could group them, and make them extend base class that will have `@classmethod`s defined, and that might save some resources.